PyAr/pyafipws
# Fixed tests from gsoc19

## Summary

fixes the issues that arose from merging the tests from PR #59. 
This is also in according to issue #6 and issue from [chazuttu/Pyafipws](https://github.com/chazuttu/pyafipws/issues/10). 

There is an update in requirements.txt file and 
cassettes are generated from recording with pytest-vcr.





## Checklist

- [x] Classes, Variables, function and methods logic  ok
- [x] Comments written explaining what the code does
- [x] All python code is PEP8 compliant (run black .)
- [x] No lint issues (run flake8)
- [x] Test coverage with pytest implemented
- [x] Reviewers assigned (at least 1 mentor)

## Manual test evidence

`$ pytest`

```
================================================== test session starts ==================================================
platform win32 -- Python 3.9.2, pytest-6.2.3, py-1.10.0, pluggy-0.13.1
rootdir: C:\Users\shiva\OneDrive\Desktop\Pyafipaws_Utkarsh\pyafipws
plugins: html-3.1.1, metadata-1.11.0, vcr-1.0.2
collected 236 items

tests\test_ws_sr_padron.py s.sx                                                                                    [  1%]
tests\test_wsaa.py .....                                                                                           [  3%]
tests\test_wsaa_crypto.py ....                                                                                     [  5%]
tests\test_wsbfev1.py ..........                                                                                   [  9%]
tests\test_wscdc.py s.......                                                                                       [ 13%]
tests\test_wsct.py ..........xx..xxxxxxxxxxssxxxxxx                                                                [ 26%]
tests\test_wsfev1.py ......                                                                                        [ 29%]
tests\test_wsfev1_dummy.py .                                                                                       [ 29%]
tests\test_wsfexv1.py ..........                                                                                   [ 33%]
tests\test_wslsp.py ......................................x.                                                       [ 50%]
tests\test_wsltv.py ............................x.                                                                 [ 63%]
tests\test_wslum.py ........................x.                                                                     [ 74%]
tests\test_wsmtx.py ..............x......x..........x                                                              [ 88%]
tests\test_wsremcarne.py ...........................                                                               [100%]

=================================================== warnings summary ==================================================== 
finaa\lib\site-packages\pysimplesoap\transport.py:141
  c:\users\shiva\onedrive\desktop\pyafipaws_utkarsh\pyafipws\finaa\lib\site-packages\pysimplesoap\transport.py:141: DeprecationWarning: inspect.getargspec() is deprecated since Python 3.0, use inspect.signature() or inspect.getfullargspec()    
    if 'timeout' in inspect.getargspec(httplib2.Http.__init__)[0]:

finaa\lib\site-packages\future\standard_library\__init__.py:65
  c:\users\shiva\onedrive\desktop\pyafipaws_utkarsh\pyafipws\finaa\lib\site-packages\future\standard_library\__init__.py:65: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses

tests/test_ws_sr_padron.py::test_consultar_a5
  c:\users\shiva\onedrive\desktop\pyafipaws_utkarsh\pyafipws\finaa\lib\site-packages\pysimplesoap\simplexml.py:49: Runtimel.py:49: RuntimeWarning: removing unsupported UTC offset
    warnings.warn('removing unsupported UTC offset', RuntimeWarning)

-- Docs: https://docs.pytest.org/en/stable/warnings.html
================================ 206 passed, 5 skipped, 25 xfailed, 3 warnings in 15.72s ================================ ===============
```
